ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/stun.j2

```diff
@@ -0,0 +1,38 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - stun #}
+{% if stun.server is arista.avd.defined %}
+
+### STUN server
+
+#### STUN Local server
+
+| local_interface |
+{%     if stus.server.local_interface is defined %}
+{{ stun.server.local_interface }}
+{%     endif %}
+
+### STUN Client
+
+#### STUN servers
+
+{%     if stun.client is arista.avd.defined %}
+| server_profile | ip_address |
+  {%         for server in stun.client %}
+    {%             if server.server_profile is arista.avd.defined %}
+    server_profile {{ server.server_profile }}
+    {%             endif %}
+    {%             if server.ip_address is arista.avd.defined %}
+    ip address {{ server.ip_address }}
+    {%             endif %}
+  {%         endfor %}
+{%     endif %}
+#### Stun device configuration
+
+```eos
+{%     include 'eos/stun.j2' %}
+```
+{% endif %}
```

Comment:
* Add a documentation section for STUN top level config
* inverted client and server regarding ordering of config
* Fixed a typo: `stus` instead of `stun`
* Uses a fully qualified name for `defined` to use the one from AVD
* Add some logic to not render the STUN server title by default if not needed.
* What is a "local server" ?
* * if there is only one server we may consider changing the table format

```suggestion
{% if stun is arista.avd.defined %}

### STUN
{%     if stun.client is arista.avd.defined %}

#### STUN Client
{%         if stun.client.server_profiles is arista.avd.defined %}
##### STUN Client server-profiles

| server_profile | ip_address |
{%             for server_profile in stun.client.server_profiles %}
| {{ server_profile.name }} | {{ server_profile.ip_address }}|   
{%             endfor %}
{%         endif %}
{%     endif %}
{%     if stun.server is arista.avd.defined %}

#### STUN server
{%         if stun.server.local_interface is arista.avd.defined %}

##### STUN Local server

| local_interface |
{{ stun.server.local_interface }}
{%         endif %}
{%     endif %}

#### STUN Device Configuration

```eos
{%     include 'eos/stun.j2' %}
```
{% endif %}
```